### PR TITLE
Update _index.en.md

### PR DESCRIPTION
**Pull request by dinger1986**

### CODE REVIEW

**Comment by rustdesk** [comment]
should get 1.2.2 with script, you have it in your update script.

**Comment by dinger1986** [comment]
changed linux script as well

File: content/client/Client Deployment/_index.en.md [hunk 1]
```diff
@@ -145,8 +145,9 @@ open -n /Applications/RustDesk.app
 ```sh
 #!/bin/bash
 
-# Assign the value "XYZ" to the password variable
-rustdesk_password="XYZ"
+# Assign a random value to the password variable
+rustdesk_password=$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 8 | head -n 1)
```
Comment (rustdesk) [inline content/client/Client Deployment/_index.en.md:149]: for what?

Comment (rustdesk) [reply content/client/Client Deployment/_index.en.md:149]: why do not leave permanent password empty?

Comment (rustdesk) [reply content/client/Client Deployment/_index.en.md:149]: Follow some configuration template file used by postgres or sshd etc.

Comment (rustdesk) [reply content/client/Client Deployment/_index.en.md:149]: It is ok if the user can get the random password somewhere.

Comment (dinger1986) [reply content/client/Client Deployment/_index.en.md:149]: they dont set passwords, this is an install script not a config file. its moreso that I have something to point people to with support. I was going to use the same template for windows, linux and mac but havent got there yet and better to have something

Comment (dinger1986) [reply content/client/Client Deployment/_index.en.md:149]: > It is ok if the user can get the random password somewhere.

its output at the end of the script, rmms, intune etc have collectors so would pick that up

Comment (rustdesk) [reply content/client/Client Deployment/_index.en.md:167]: Understand now

Comment (dinger1986) [reply content/client/Client Deployment/_index.en.md:167]: have updated the description a bit to make it more obvious what its installed for, once I am home again I will do some video walk throughs

Comment (rustdesk) [reply content/client/Client Deployment/_index.en.md:167]: Great

**Pull request merged**